flint.ui/src/components/Datepicker/DatepickerPoint.vue

```diff
@@ -17,6 +25,9 @@
 import moment from 'moment'
 
 export default {
+  props: {
+    datepickerWidth: String
```

Comment:
We can just rename `datePickerWidth` as `size`.
Also, it would be better if we can have:
- Default Prop for size (medium)
- Validator function to check if the value is from ["small", "medium", "large"] (refer Vue3 docs for Props)

Reply:
Sure @sohamsshah Thanks for the suggestions, I will fix these issues and add the listed features 👍🏽

Reply:
Hey @sohamsshah I have done the listed changes, you can review this PR now 👍🏽

Reply:
Is the Validator function also added?